js/main.js

```diff
@@ -74,4 +74,4 @@ const app = new Vue({
 
 setInterval(() => {
   app.$store.dispatch('init')
-}, 5 * 1000)// 5 seconds
+}, 2 * 1000)// 5 seconds
```

Comment:
Jak zmieniasz to, to też zmieniaj komentarz obok.

Reply:
Fak nie zobaczyłem, oks.

Reply:
Gotowe